sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/DefaultOfferRequirementProvider.java

```diff
@@ -30,9 +30,11 @@
     private final StateStore stateStore;
     private final UUID targetConfigurationId;
 
-    public DefaultOfferRequirementProvider(
-            StateStore stateStore,
-            UUID targetConfigurationId) {
+    /**
+     * Creates a new instance which relies on the provided {@link StateStore} for storing known tasks, and which expects
```

Comment:
Not clear what "expects" means here? It will tag tasks with that configuration ID, but it doesn't really _expect_ it as far as I can tell.